src/charm.py

```diff
@@ -418,6 +419,26 @@ def _report_service_ips(self):
             x.spec.cluster_ip for x in services.items if x.metadata.name in all_k8s_service_names
         ]
 
+    def _report_ingress_ips(self) -> list:
+        """Report on ingress IP(s)."""
```

Comment:
We have a standard for docstrings that includes things like describing the return value, do you think it applies here? https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#docstrings